packages/react/src/components/InlineAlert/index.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withKnobs, select, text } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+import { withInfo } from "@storybook/addon-info";
+
+import InlineAlert from "./";
+import { INLINE_ALERT_TYPES } from "../../constants/";
+
+storiesOf("Alerts")
+  .addDecorator(withKnobs)
+  .add(
+    "InlineAlert",
+    withInfo("")(() => (
```

Comment:
So this knob if for displaying the comments in the component, right?

Reply:
That's a nice story :)

So this know (when passing a readme file here) adds the readme to "Show info" page of a component. Meaning that the Readme would be available also from the storybook itself, not only within the repository.

But they've recently discovered a bug where it fails to parse triple `\`` that we use to paste code blocks.

So the bug is reportedly fixed and it's in beta now I believe. So I'm temporarily passing an empty string there, waiting for the bugfix to become public, and then we'll upgrade the version of the knob and pass Readmes there :)

Reply:
👍